crnk-core/src/main/java/io/crnk/core/engine/internal/jackson/ObjectLinkErrorDataSerializer.java

```diff
@@ -0,0 +1,24 @@
+package io.crnk.core.engine.internal.jackson;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import io.crnk.core.engine.document.ErrorData;
+
+/**
+ * @author AdNovum Informatik AG
+ */
+public class ObjectLinkErrorDataSerializer extends ErrorDataSerializer {
```

Comment:
I noticed that at a number of locations with now have to implementations, one for primitive links and ones for object links. Potentially it would rather make sense to introduce an abstraction for writing links? then a single implementation at the various locations like error data and ObjectLinkResourceMapper would suffice.